docs/parts/physical_systems/electric_motors/eesm.rst

```diff
@@ -0,0 +1,22 @@
+Extertnally Excited Synchronous Motor
+######################################
+
+Schematic
+*********
+
+
+Electrical ODE
+**************
+
+
+
+Torque Equation
+***************
+
+
+Code Documentation
+******************
+
+.. autoclass:: gym_electric_motor.physical_systems.electric_motors.PermanentMagnetSynchronousMotor
+   :members:
+   :inherited-members:
```

Comment:
Is this to be done in a later update or do you need any help with this?

Also, the entry behind "autoclass" is from the PMSM.